apps/server/src/modules/tool/external-tool/service/datasheet-pdf.service.spec.ts

```diff
@@ -58,6 +60,7 @@ describe(DatasheetPdfService.name, () => {
 		describe('when tool is oauth2 tool with optional properties and custom parameters', () => {
 			const setup = () => {
 				const user: UserDO = userDoFactory.buildWithId();
+				const school: School = schoolFactory.build();
```

Comment:
well, you didn't need a whole school just to mock a name but ok

Reply:
yeah... you're right ^^ -> all done :)